FND/DatesAndTimes/BusinessDates.rdf

```diff
@@ -75,54 +76,54 @@ They are modularized this way to minimize the ontological committments that are
 	</owl:NamedIndividual>
 	
 	<owl:NamedIndividual rdf:about="&fibo-fnd-dt-bd;BusinessDayModifiedFollowing">
 		<rdf:type rdf:resource="&fibo-fnd-dt-bd;BusinessDayConvention"/>
 		<rdfs:label>business day modified following</rdfs:label>
 		<sm:normativeReference>FPML 5.1 &quot;BusinessDayConventionEnum&quot;</sm:normativeReference>
-		<skos:definition>BusinessDayModifiedFollowing is a BusinessDayConvention that means a non-business date will be adjusted to the first following day that is a business day unless that day falls in the next calendar month, in which case that date will be the first preceding day that is a calendar date.</skos:definition>
+		<skos:definition>convention specifying that a non-business date will be adjusted to the first following day that is a business day unless that day falls in the next calendar month, in which case that date will be the first preceding day that is a calendar date</skos:definition>
 	</owl:NamedIndividual>
 	
 	<owl:NamedIndividual rdf:about="&fibo-fnd-dt-bd;BusinessDayModifiedPreceding">
 		<rdf:type rdf:resource="&fibo-fnd-dt-bd;BusinessDayConvention"/>
 		<rdfs:label>business day modified preceding</rdfs:label>
 		<sm:normativeReference>FPML 5.1 &quot;BusinessDayConventionEnum&quot;</sm:normativeReference>
-		<skos:definition>BusinessDayModifiedPreceding is a BusinessDayConvention that means that a non-business date will be adjusted to the first preceding day that is a business day unless that day falls in the previous month, in which case that date will be the first following day that is a business day.</skos:definition>
+		<skos:definition>convention specifying that a non-business date will be adjusted to the first preceding day that is a business day unless that day falls in the previous month, in which case that date will be the first following day that is a business day</skos:definition>
 	</owl:NamedIndividual>
 	
 	<owl:NamedIndividual rdf:about="&fibo-fnd-dt-bd;BusinessDayNearest">
 		<rdf:type rdf:resource="&fibo-fnd-dt-bd;BusinessDayConvention"/>
 		<rdfs:label>business day nearest</rdfs:label>
 		<sm:normativeReference>FPML 5.1 &quot;BusinessDayConventionEnum&quot;</sm:normativeReference>
-		<skos:definition>BusinessDayNearest is a BusinessDayConvention that means that a non-business date will be adjusted to the nearest day that is a business day -- i.e. if the non-business day falls on any day other than a Sunday or a Monday, it will be the first preceding day that is a business day, and will be the first following business day if it falls on a Sunday or a Monday.</skos:definition>
+		<skos:definition>convention specifying that a non-business date will be adjusted to the nearest day that is a business day -- i.e. if the non-business day falls on any day other than a Sunday or a Monday, it will be the first preceding day that is a business day, and will be the first following business day if it falls on a Sunday or a Monday</skos:definition>
```

Comment:
"first preceding" could be taken as "earliest preceding". "latest preceding would be clearer".
The algorithm is unnecessarily specific and wrong - in the case of the Saturday after Thanksgiving (assuming the Friday is not a business day) then the nearest business day will be the following Monday but the algorithm says the preceding Wednesday.

Reply:
Let's address this under a new issue - I only normalized the definition, did not change the content.